src/main/java/com/netflix/imflibrary/st2067_100/OutputProfileListModel_st2067_100_2014.java

```diff
@@ -226,22 +226,22 @@ private PixelEncoderMacro createPixelEncoderMacro(PixelEncoderType pixelEncoderT
                 input, output);
     }
 
-    private AudioRoutingMixingMacro createAudioRoutingMixingMacro(AudioRoutingMixingMacroType audioRoutingMixingMacroType) {
-        if(audioRoutingMixingMacroType.getOutputEntityList() == null) {
+    private AudioRoutingMixingMacro createAudioRoutingMixingMacro(AudioRoutingMixingMacroType AudioRoutingMixingMacroType) {
```

Comment:
@palemieux - I don't follow why the parameter names should be capitalized, can you explain?

Reply:
That was a mistake resulting from a search and replace that was too zealous.

Fixed at https://github.com/sandflow/photon/tree/issues/correct-capitalization

Reply:
thanks!